include/net/nrf_cloud.h

```diff
@@ -589,18 +587,6 @@ int nrf_cloud_connect(const struct nrf_cloud_connect_param *param);
  */
 int nrf_cloud_sensor_data_send(const struct nrf_cloud_sensor_data *param);
 
-/**
- * @brief Update the device shadow with sensor data.
- *
- * @param[in] param Sensor data; the data pointed to by param->data.ptr must be a
- *                  valid JSON string.
- *
- * @retval 0       If successful.
- * @retval -EACCES Cloud connection is not established; wait for @ref NRF_CLOUD_EVT_READY.
- *                 Otherwise, a (negative) error code is returned.
- */
-int nrf_cloud_shadow_update(const struct nrf_cloud_sensor_data *param);
```

Comment:
could you put removing of `nrf_cloud_shadow_update` in a separate commit since it is removing a function from the library?  this will also require a [changelog entry:](https://github.com/nrfconnect/sdk-nrf/blob/main/doc/nrf/releases/release-notes-changelog.rst)